week4/이지영/가장큰수.js

```diff
@@ -0,0 +1,18 @@
+function solution(numbers) {
+  let answer = "";
+  let arr = [];
+  for (let i = 0; i < numbers.length; i++) {
+    arr.push(numbers[i].toString()); // 문자열로 변환하여 배열에 넣기
+  }
+  arr.sort((a, b) => b + a - (a + b)); // 문자열을 합쳐서 비교하기
```

Comment:
imo: 이 부분은 map으로 표현하는 것이 더 좋을 것 같습니다. 이유는 값을 변경하는 경우 "복사된" 자료구조에 반영하는게 사이드 이펙트를 줄여줄 수 있을거라 생각했어요!

```suggestion
  const copyed = number.map(...).sort(..)
```